src/components/animated-menu-button/animated-menu-button.js

```diff
@@ -82,7 +82,7 @@ class AnimatedMenuButton extends React.Component {
 
   static defaultProps = {
     direction: 'left',
-    size:      'medium'
+    size: 'medium'
```

Comment:
I don't  think we're enforcing one standard - we've turned off the linting rule for this.